### PR TITLE
Add blobList as a member, and separate out sparseList test

### PR DESCRIPTION
**Pull request by drganjoo**
#### Background
* What do these changes do? 
1. Move sparseList/Map test into a separate operation. 
2. Add blobList as a member field.
* Why are they important?
To ensure blob lists can be serialized / deserialized correctly.

#### Testing
* How did you test these changes?

### CODE REVIEW

File: smithy-aws-protocol-tests/model/rpcV2/cbor-structs.smithy [hunk 1]
```diff
@@ -335,7 +335,28 @@ use smithy.test#httpResponseTests
             doubleValue: "-Infinity",
             floatValue: "-Infinity"
         }
-    }
+    },
+        {
+        id: "RpcV2CborSupportsUpcastingDataOnDeserialize",
+        protocol: rpcv2Cbor,
+        documentation: "Supports upcasting from a smaller byte representation of the same date type.",
```
Comment (kstich) [inline smithy-aws-protocol-tests/model/rpcV2/cbor-structs.smithy:342]:
```suggestion
        documentation: "Supports upcasting from a smaller byte representation of the same data type.",
```

File: smithy-aws-protocol-tests/model/rpcV2/cbor-lists.smithy [hunk 2]
```diff
@@ -309,3 +329,94 @@ structure StructureListMember {
...
+    }
+])
+operation SparseNullsOperation {
+    input: SparseNullsOperationInputOutput
+    output: SparseNullsOperationInputOutput
+}
+
+structure SparseNullsOperationInputOutput {
+    sparseStringList: SparseStringList
+    sparseStringMap: SparseStringMap
```
Comment (kstich) [inline smithy-aws-protocol-tests/model/rpcV2/cbor-lists.smithy:421]: The tests for sparse maps that are currently in the `cbor-maps.smithy` file should be pulled out to this operation as well.

**Review by kstich — CHANGES_REQUESTED** [review]

File: smithy-aws-protocol-tests/model/rpcV2/cbor-structs.smithy (hunk 1 above)
Comment (drganjoo) [reply smithy-aws-protocol-tests/model/rpcV2/cbor-structs.smithy]: Fixed in both upcasting tests.

File: smithy-aws-protocol-tests/model/rpcV2/cbor-lists.smithy (hunk 2 above)
Comment (drganjoo) [reply smithy-aws-protocol-tests/model/rpcV2/cbor-lists.smithy:421]: Have moved them into a separate operation

File: smithy-aws-protocol-tests/model/rpcV2/empty-input-output.smithy [hunk 3]
```diff
@@ -83,29 +83,29 @@ use smithy.test#httpResponseTests
...
+        bodyMediaType: "application/cbor",
+        body: "v/8=",
         appliesTo: "client",
     },
     {
-        id: "no_input_client_allows_empty_cbor",
+        id: "NoOutputClientAllowsEmptyBody",
         protocol: rpcv2Cbor,
-        documentation: "Client should accept CBOR empty struct if no output",
+        documentation: "Client should accept empty body if no output",
```
Comment (kstich) [inline smithy-aws-protocol-tests/model/rpcV2/empty-input-output.smithy:101]: The difference between this test and `no_output` is the `Content-Type` header, let's call that out here.

**Review by kstich — CHANGES_REQUESTED** [review]

File: smithy-aws-protocol-tests/model/rpcV2/empty-input-output.smithy [hunk 4]
```diff
@@ -53,17 +53,17 @@ use smithy.test#httpResponseTests
         appliesTo: "server"
     },
     {
-        id: "NoInputServerIngoresUnexpectedFields",
```
[the remark this answers is not in the document]
Comment (drganjoo) [reply smithy-aws-protocol-tests/model/rpcV2/empty-input-output.smithy:56]: Oh yes. I think let's remove `no_input_sever_allows_accept` and keep this.

**Review by kstich — APPROVED** [review]

**Pull request merged**